src/fetchtastic/setup_config.py

```
@@ -354,7 +359,11 @@ def is_fetchtastic_installed_via_pip() -> bool:
     try:
         # Check if fetchtastic is in pip list
         result = subprocess.run(
-            ["pip", "list"], capture_output=True, text=True, check=False
+            ["pip", "list"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)